scripts/install-worker.sh

```diff
@@ -82,28 +82,6 @@ sudo yum versionlock kernel-$(uname -r)
 # Remove the ec2-net-utils package, if it's installed. This package interferes with the route setup on the instance.
 if yum list installed | grep ec2-net-utils; then sudo yum remove ec2-net-utils -y -q; fi
 
-################################################################################
-### Time #######################################################################
-################################################################################
-
-# Make sure Amazon Time Sync Service starts on boot.
-sudo chkconfig chronyd on
-
-# Make sure that chronyd syncs RTC clock to the kernel.
-cat << EOF | sudo tee -a /etc/chrony.conf
-# This directive enables kernel synchronisation (every 11 minutes) of the
-# real-time clock. Note that it can’t be used along with the 'rtcfile' directive.
-rtcsync
-EOF
```

Comment:
I verified that this is already present in `/etc/chrony.conf` in the latest AL2 minimal AMI. We should not configure `chrony` here to avoid diverging from the defaults provided by AL2.